pkg/api/vm/formatter.go

```diff
@@ -103,11 +105,6 @@ func (vf *vmformatter) formatter(request *types.APIRequest, resource *types.RawR
 		resource.AddAction(request, unpauseVM)
 	}
 
-	if canMigrate(vmi) {
```

Comment:
`migrate` and `abortMigration` menu are mutually exclusive, or both are not existing (e.g. vmi is not in running state)

after this PR, they may come together, did you notice when testing?